react-native-gutenberg-bridge/ios/RNReactNativeGutenbergBridge.m

```diff
@@ -15,6 +15,8 @@ @interface RCT_EXTERN_MODULE(RNReactNativeGutenbergBridge, NSObject)
 RCT_EXTERN_METHOD(editorDidMount:(NSArray *)unsupportedBlockNames)
 RCT_EXTERN_METHOD(editorDidEmitLog:(NSString *)message logLevel:(int)logLevel)
 RCT_EXTERN_METHOD(editorDidAutosave)
+RCT_EXTERN_METHOD(requestImageFullscreenPreview:(NSString *)mediaUrl)
 RCT_EXTERN_METHOD(fetchRequest:(NSString *)path resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+RCT_EXTERN_METHOD(requestImageFullscreenPreview:(NSString *)mediaUrl)
```

Comment:
This looks like a merge problem, the `requestImageFullscreenPreview` is being added twice.

Reply:
Thanks, fixed, https://github.com/wordpress-mobile/gutenberg-mobile/pull/1597/commits/2b5ecb31cc9f4633f15f0545194b88d9fe1ba438